src/main/java/org/quantumbadger/redreader/activities/BaseActivity.java

```diff
@@ -79,6 +84,18 @@ else if (orientation == PrefsUtility.ScreenOrientation.LANDSCAPE)
 			setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE);
 	}
 
+	private void setTorFromPrefs(boolean force) {
+		tor = PrefsUtility.network_tor(this, sharedPreferences);
+		if (tor) {
+			if (OrbotHelper.isOrbotInstalled(this)){
```

Comment:
I think it would be better to check this in the preference fragment, and to not allow the user to select the preference if Orbot isn't installed (a dialog can be popped up to tell them why).